.github/workflows/on.pull-request.main.yaml

```diff
@@ -28,13 +28,13 @@ jobs:
       fail-fast: false
       matrix:
         service:
-         - s3
-         - ecr
-         - sns
-         - sqs
-         - elasticache
-         - dynamodb
-         - apigatewayv2
+          - s3
+          - ecr
+          - sns
+          - sqs
+          - elasticache
+          - dynamodb
+          - apigatewayv2
```

Comment:
did you IDE or editor auto-change the above? :)

Reply:
Yes. Had the auto-format enabled. Synced up with previous version.